src/main/java/com/guysfromusa/carsgame/entities/Map.java

```diff
@@ -0,0 +1,43 @@
+package com.guysfromusa.carsgame.entities;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+@Entity
+public class Map {
```

Comment:
Map name would be misleading with object Type Map. Please consider it.

Reply:
Yes you're right, this initial class was for skeleton purpose, we can think a little bit about better name my proposal is: StreetMap, Robert will be responsible for Map API so he will use new name